SR2021/2020-10-29-sr2021-announcement.md

```diff
@@ -0,0 +1,26 @@
+---
+to: Potential team leaders
+subject: Announcing Student Robotics 2021
+---
+
+Hello there!
+
+We're pleased to announce sign-ups are now open for Student Robotics 2021!
+
+## Competition
+
+Unfortunately, we are unable to find a way to run a physical competition in a COVID safe way. Instead, following on from the success of our [previous virtual competition](https://studentrobotics.org/news/2020-07-25-post-competition/), we will be running the competition in a simulated environment. The competition's time scale will be very similar to what you're used to, with just a few short weeks until we'll kick off the competition with our Kickstart event. Teams will then have roughly 7 months to build their robot before the final competition on 1st May.
+
+However, on top of this we will be running monthly league sessions. These league sessions will be similar to those run for the virtual competition, and much like the regular competition league will seed your position in the final knockout.
+
+## Kickstart
+
+Kickstart will be streamed live on **21st November 2020**. Here we'll announce the game for this year, go into further details on how the competition year will work, and of course answer any questions you have.
+
+Visit our website for more information.
```

Comment:
If https://github.com/srobo/website/pull/231 lands before this email is sent we should add a link.